src/pages/events/eventspage.jsx

```diff
@@ -12,29 +12,29 @@ function Events() {
 
 
 //Snap Scroll on buttons
-// const firstArrow = document.querySelector(".Arrowicon")
-// const firstCard = document.querySelector(".left-card1")
-// firstArrow.addEventListener("click", () =>{
-//   firstCard.scrollIntoView({behavior: "smooth"})
-// })
-
-// const seccondArrow = document.querySelector(".left-card1-arrow")
-// const seccondCard = document.querySelector(".right-card1")
-// seccondArrow.addEventListener("click", () =>{
-//   seccondCard.scrollIntoView({behavior: "smooth"})
-// })
-
-// const thirdArrow = document.querySelector(".right-card1-arrow")
-// const thirdCard = document.querySelector(".left-card2")
-// thirdArrow.addEventListener("click", () =>{
-//   thirdCard.scrollIntoView({behavior: "smooth"})
-// })
-
-// const fourthArrow = document.querySelector(".left-card2-arrow")
-// const fourthCard = document.querySelector(".right-card2")
-// fourthArrow.addEventListener("click", () =>{
-//   fourthCard.scrollIntoView({behavior: "smooth"})
-// })
+const firstArrow = document.querySelector(".Arrowicon")
+const firstCard = document.querySelector(".left-card1")
+firstArrow.addEventListener("click", () =>{
```

Comment:
This line seems to break the app:
![image](https://user-images.githubusercontent.com/37623357/197524323-1914db00-e14e-47db-a79c-a60e3cbca868.png)

I'd suggest changing all the `addEventListener` calls to [`onClick` handlers in the jsx](https://reactjs.org/docs/handling-events.html).